client/evm-tracing/src/formatters/call_tracer.rs

```diff
@@ -105,6 +114,7 @@ impl super::ResponseFormatter for Formatter {
 							}
 						},
 						calls: Vec::new(),
+						//logs: Vec::new(),
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/moonbeam-foundation/moonbeam/pull/2897/commits/ec4c5ed2eb63e5a1af918b8d8df6b653843bfa4e